src/ansiblelint/utils.py

```diff
@@ -275,110 +280,169 @@ def template(
     return value
 
 
-def _include_children(
-    basedir: str,
-    k: str,
-    v: Any,
-    parent_type: FileType,
-) -> list[Lintable]:
-    # handle special case include_tasks: name=filename.yml
-    if k in INCLUSION_ACTION_NAMES and isinstance(v, dict) and "file" in v:
-        v = v["file"]
-
-    # we cannot really parse any jinja2 in includes, so we ignore them
-    if not v or "{{" in v:
-        return []
-
-    if "import_playbook" in k and COLLECTION_PLAY_RE.match(v):
-        # Any import_playbooks from collections should be ignored as ansible
-        # own syntax check will handle them.
-        return []
-
-    # handle include: filename.yml tags=blah
-    # pylint: disable=unused-variable
-    (command, args, kwargs) = tokenize(f"{k}: {v}")
+@dataclass
```

Comment:
A dataclass with code... not its supposed use but I guess that you wanted to make use of the fancy syntax provided by them. OK.